python/samples/kernel-syntax-examples/setup_logging.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft. All rights reserved.
+import logging
+
+import semantic_kernel as sk
+from semantic_kernel.connectors.ai.open_ai import OpenAIChatCompletion
+
+
+def setup_logging():
```

Comment:
I understand this is simply an example to show how one could set up their logging; however, do we need to document that if someone wants to configure the logging, they'd need to essentially call a method like this from SK's `__int__.py` so that it does set up the logging properly?

Reply:
They wouldn't need to call it from SK's `__init__.py`, right? They can set up the logging from their own scripts before importing SK.

Reply:
I didn't ask my question properly, apologies. I think it would be useful if we configured logging automatically, via the __init__.py, and then users could toggle the verbosity (or off). But we can discuss that later.

Reply:
Oh! Makes sense but I agree we can look into that in a follow up PR.